homeassistant/components/saj/sensor.py

```diff
@@ -119,61 +177,59 @@ async def async_saj():
 
         return values
 
-    def start_update_interval(event):
+    def start_update_interval(self, _):
```

Comment:
When adding the integration or restarting HA the sensors stay in an unknown state until the interval time has been reached.
It would be nice if the sensors immediately have the retrieved values and from then on schedules updates every X seconds.